src/Components/Home.js

```diff
@@ -0,0 +1,80 @@
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { useNavigate } from 'react-router-dom';
+import { BsArrowRightCircle } from 'react-icons/bs';
+import Header from './Header';
+
+import { fetchDataApi } from '../Redux/Data/data';
+
+const Home = () => {
+  const data = useSelector((state) => Object.values(state.covidData)[1]);
+  let global;
+  let countries;
+
+  if (data) {
+    [, , global] = Object.values(data);
+    [, , , countries] = Object.values(data);
+  }
+
+  const Navigate = useNavigate();
+  const dispatch = useDispatch();
+  const condition = true;
+
+  useEffect(() => {
+    dispatch(fetchDataApi());
+  }, [dispatch]);
+
+  return (
```

Comment:
- [ ] I kindly suggest that you ensure that your Homepage has a filter e.g search bar or dropdown list to filter the list of items displayed in your homepage